packages/client/screens/MusicPlaylistEditorCreationForm/__tests__/CreateMpeRoom.test.tsx

```diff
@@ -12,10 +12,10 @@ import {
 } from '../../../tests/tests-utils';
 import {
     db,
-    generateMpeWorkflowState,
     generateMpeWorkflowStateWithUserRelatedInformation,
 } from '../../../tests/data';
 import { serverSocket } from '../../../services/websockets';
+import { msToTime } from '../../MusicPlaylistEditorRoomScreen';
```

Comment:
To do not break black-box testing principles, I think we should not rely on functions we are testing to determine what we are expecting.

Reply:
you're right ! about to fix that point